### PR TITLE
(Feature) Improve jobs logging

### LINKED ISSUE #72: Improve logging
Right now the logs are not complete clear, more information could be added in order to improve debugging.

### PR DESCRIPTION
**Pull request by Agupane**
Closes #72

### CODE REVIEW

File: server/jobs/running-bot-telegram.js [hunk 1]
```diff
@@ -178,11 +182,11 @@ ${body}`,
           }
         }
       )
-      console.log(`[Telegram bot] - Telegram sended to chatId ${msg.chat.id} successfully`)
+      console.log(`[Telegram bot] - Telegram sended to chatId ${telegramChatId} successfully`)
     } catch (e) {
       console.error(e)
       bot.sendMessage(
-        msg.chat.id,
+        telegramBotKey,
```
Comment (mariano-aguero) [inline server/jobs/running-bot-telegram.js]: Req: This should be `telegramChatId`

File: server/jobs/running-bot-telegram.js [hunk 2]
```diff
@@ -191,31 +195,33 @@ ${body}`,
...
         },
         parse_mode: 'HTML'
       })
-      console.log(`[Telegram bot] - Telegram sended to chatId ${msg.chat.id} successfully`)
+      console.log(`[Telegram bot] - Telegram sended to chatId ${telegramChatId} successfully`)
     } catch (e) {
       console.error(e)
       bot.sendMessage(
-        msg.chat.id,
+        telegramBotKey,
```
Comment (mariano-aguero) [inline server/jobs/running-bot-telegram.js]: Req: This should be `telegramChatId`

File: server/jobs/running-bot-telegram.js (hunk 1 above)
Comment (Agupane) [reply server/jobs/running-bot-telegram.js]: It's done

File: server/jobs/running-bot-telegram.js (hunk 2 above)
Comment (Agupane) [reply server/jobs/running-bot-telegram.js]: It's done

**Pull request merged**